rest_framework/renderers.py

```diff
@@ -492,16 +508,21 @@ def get_rendered_html_form(self, data, view, method, request):
 
             if has_serializer:
                 if method in ('PUT', 'PATCH'):
-                    serializer = view.get_serializer(instance=instance, **kwargs)
+                    serializer = view.get_serializer(instance=instance,
+                                                     **kwargs)
                 else:
                     serializer = view.get_serializer(**kwargs)
             else:
                 # at this point we must have a serializer_class
                 if method in ('PUT', 'PATCH'):
-                    serializer = self._get_serializer(view.serializer_class, view,
-                                                      request, instance=instance, **kwargs)
+                    serializer = self._get_serializer(view.serializer_class,
```

Comment:
Not convinced by this indentation style. If we must push arguments onto a newline, let's stick with ...

    something(
        arg1,
        arg2
     )

Seems far more readable to me than the "push everything up to the initial bracket" style